mapswipe_workers/Dockerfile

```diff
@@ -2,41 +2,36 @@
 # Image includes python3.6, gdal-python, gdal-bin
 FROM osgeo/gdal:ubuntu-small-latest
 
+# Set C.UTF-8 locale as default (Needed by the Click library)
 ENV LC_ALL=C.UTF-8
 ENV LANG=C.UTF-8
 
 # Install pip
 RUN apt-get update
 RUN apt-get --yes install python3-pip
 
-# Create directories for config, logs and data
-ARG config_dir=/usr/share/config/mapswipe_workers/
-ARG repo_dir=/usr/local/mapswipe_workers/
-ARG data_dir=/var/lib/mapswipe_workers/
-ARG logs_dir=/var/log/mapswipe_workers/
-
-RUN mkdir -p $config_dir
-RUN mkdir -p $repo_dir
-RUN mkdir -p $logs_dir
-RUN mkdir -p $data_dir
-RUN mkdir -p $data_dir"api-data"
-RUN mkdir -p $data_dir"api-data/agg_results/"
-RUN mkdir -p $data_dir"api-data/groups/"
-RUN mkdir -p $data_dir"api-data/history/"
-RUN mkdir -p $data_dir"api-data/projects/"
-RUN mkdir -p $data_dir"api-data/results/"
-RUN mkdir -p $data_dir"api-data/tasks/"
-RUN mkdir -p $data_dir"api-data/yes_maybe/"
-RUN mkdir -p $data_dir"api-data/hot_tm/"
+# Create user mapswipe
+RUN groupadd --system mapswipe
+RUN useradd --create-home --no-log-init --system --gid mapswipe mapswipe
+
+# Create directory for logs and make it accsesably for the mapswipe user
+RUN mkdir --parents /var/log/mapswipe_workers
+RUN chown mapswipe_workers: /var/log/mapswipe_workers/
```

Comment:
I think this line should be:
`RUN chown mapswipe:mapswipe /var/log/mapswipe_workers/`

Reply:
Yes your are right. Thanks for pointing out.